### ISSUE TITLE
tuple(a, b, ...etc) always returns first element, instead of tuple

### ISSUE THREAD

**Issue by larskinn** [AlexGalays/spacelift#16]
### Problem
```
> tuple(1, 2)
=> 1
```
expected `[1, 2]`, but got `1`

```
> tuple("first", "second", "third")
=> "first"
```
expected `["first", "second", "third"]`, but got `"first"`

### Solution
I think this code
```ts
export function tuple(arr: any[]): any {
  return arr
}
```

... just needs to be changed to

```ts
export function tuple(...arr: any[]): any {
  return arr
}
```

**Comment by AlexGalays** [comment]
Wow, guess that's what happens when you don't add a test!

**Comment by AlexGalays** [comment]
Fixed in `0.8.7`; thanks for the report!
